app/controllers/admin/schedules_controller.rb

```diff
@@ -9,7 +9,9 @@ class SchedulesController < Admin::BaseController
     load_resource :selected_schedule, through: :program, singleton: true
     load_resource :venue, through: :conference, singleton: true
 
-    def index; end
+    def index 
+      @rooms = @conference.venue.rooms if @conference.venue
```

Comment:
In the index file we need to check if the are any rooms (true or false), we don't need to have the whole collection of rooms available.